contracts/index/DepositZap.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: BUSDL-1.1
+pragma solidity 0.6.11;
```

Comment:
Thoughts on doing that upgrade to one of the 0.8.x versions? I believe the default for hardhat now is 0.8.9.